src/main/java/frc/robot/pose/FieldAprilTagCollection.java

```diff
@@ -0,0 +1,46 @@
+package frc.robot.pose;
+
+import frc.robot.RobotConstants;
+import java.util.NoSuchElementException;
+
+/**
+ *
+ * @author dcowden
+ */
+public class FieldAprilTagCollection {
+
+    public interface AprilTagIds {
+        public static int RED_RIGHT = 1;
+        public static int RED_MIDDLE =2;
+        public static int RED_LEFT = 3;
+        public static int RED_LOADING=4;
+        public static int BLUE_LOADING=5;
+        public static int BLUE_RIGHT=6;
+        public static int BLUE_MIDDLE=7;
+        public static int BLUE_LEFT=8;
+    }
+
+    public static final FieldAprilTag RED_RIGHT = new FieldAprilTag(0.0, 0.0, 0.0, AprilTagIds.RED_RIGHT,  RobotConstants.TEAM.RED);
```

Comment:
we need to populate these values. Also, this code has way too much static stuff. Ideally there' would be a better way but i'm not sure what it is